fit-sarix-models/cache_data.R

```diff
@@ -322,8 +322,21 @@ required_locations <-
 # even if we are delayed and create it Tuesday morning.
 reference_date <- as.character(lubridate::floor_date(Sys.Date(), unit = "week") + 1)
 
-# Load data
-hosp_data <- load_hosp_data(pathogen = "covid", as_of = reference_date)
+# Load data (by dropping the last observation)
+hosp_data <- covidData::load_data(
+  spatial_resolution = c("national", "state"),
+  temporal_resolution = "daily",
+  measure = "hospitalizations",
+  drop_last_date = TRUE
+) %>%
+  dplyr::left_join(covidData::fips_codes, by = "location") %>%
+  dplyr::transmute(
+    date,
+    location,
+    location_name = ifelse(location_name == "United States", "US", location_name),
+    value = inc) %>%
+  dplyr::filter(location != "60") %>%
```

Comment:
Let's drop this filter -- we are able to produce forecasts for location "60"
```suggestion
```

Reply:
need this filter to calculate 'state_pop100k' in `run_all_models_one_date_by_loc.py`